lib/node_modules/@stdlib/constants/float32/sqrt-three/include/stdlib/constants/float64/sqrt_three.h

```diff
@@ -0,0 +1,27 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef STDLIB_CONSTANTS_FLOAT32_SQRT_THREE_H
+#define STDLIB_CONSTANTS_FLOAT32_SQRT_THREE_H
+
+/**
+* Macro for the square root of 3 as a single-precision floating-point number.
+*/
+#define STDLIB_CONSTANT_FLOAT32_SQRT3 1.7320507764816284
```

Comment:
```suggestion
#define STDLIB_CONSTANT_FLOAT32_SQRT3 1.7320507764816284f
```

You need to include the suffix for single-precision.